package com.upskill.exception;

import java.util.Scanner;

public class ArithmaticExceptionExample {
	
	public static void main(String[] args) {
		
		Scanner sc = new Scanner(System.in);
		
		try {
			System.out.println("Enter the two numbers");
			int a = sc.nextInt();
			int b= sc.nextInt();
			
			System.out.println("Division is "+ a/b);
			
		}catch (ArithmeticException e) {
			System.out.println(e.getMessage() + "Exception occured");
			e.printStackTrace();
			
		}finally {
			
		}
		
		
	}
	
}
